UI/multiview.cpp

```diff
@@ -174,8 +174,7 @@ void Multiview::Update(MultiviewLayout multiviewLayout, bool drawLabel,
 		multiviewScenes.emplace_back(OBSGetWeakRef(src));
 		obs_source_inc_showing(src);
 
-		std::string name = std::to_string(numSrcs) + " - " +
-				   obs_source_get_name(src);
+		std::string name = obs_source_get_name(src);
 		multiviewLabels.emplace_back(CreateLabel(name.c_str(), h / 3));
```

Comment:
This could just use `CreateLabel(obs_source_get_name(src), ...)` without the detour via the std string now

Reply:
Done